tb_plugin/torch_tb_profiler/io/cache.py

```diff
@@ -44,11 +45,14 @@ def read(self, filename):
         local_file = self.get_remote_cache(filename)
         return read(local_file)
 
+    def get_cache_dir(self):
```

Comment:
https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters

Reply:
seems we can directly access the attribute

Reply:
yep, fixed